lib/pure/os.nim

```diff
@@ -1095,6 +1095,8 @@ type
     creationTime*: times.Time         ## Time file was created. Not supported on all systems!
     blockSize*: int                   ## Preferred I/O block size for this object.
                                       ## In some filesystems, this may vary from file to file.
+    isRegular*: bool                  ## Is file regular? (on Unix some "files"
+                                      ## can be non-regular like FIFOs, devices)
 
 template rawToFormalFileInfo(rawInfo, path, formalInfo): untyped =
```

Comment:
- On \*nix, are symlinks, etc. considered "regular files"?
- Windows needs to be handled. If the above question is `true`, then `isRegular` can just always be set to `true`.
- If it's not too much trouble, I think it's time for `rawToFormalFileInfo` to be split into 2 different templates.

Reply:
> On *nix, are symlinks, etc. considered "regular files"?

Not really, they are a separate type of file (even links to directories).  But here `isRegular` can be both `true` or `false` depending on target. I'll re-read documentation.

> Windows needs to be handled. If the above question is true, then isRegular can just always be set to true.

Thank you. May be an omission, I'll check that.